modules/tinymce/src/core/test/ts/browser/EditorUploadTest.ts

```diff
@@ -293,6 +293,34 @@ UnitTest.asynctest('browser.tinymce.core.EditorUploadTest', (success, failure) =
       Assert.eq('Status is false', result[0].status, false);
       Assert.eq('Uri is empty', result[0].uploadUri, '');
       Assert.eq('Suitable number of stacks added', 2, editor.undoManager.data.length);
+      Assert.eq('Test', [
```

Comment:
This seems like it's testing a lot of the same things as the previous assertions, just in more detail. Are you able to either (1) remove the piecewise asserts and just keep this big one or (2) break this up into a bunch more of the piecewise asserts that assert the sections of this object that you're actually worried about changing?

Unless you want the test to depend on every field of the object,I think I'd kind of prefer 2 a little bit, but I don't really have a strong preference either way.